src/sparsezoo/model/model.py

```diff
@@ -78,12 +79,14 @@ def __init__(self, source: str, download_path: Optional[str] = None):
 
         if self.source.startswith(ZOO_STUB_PREFIX):
             # initializing the files and params from the stub
-            files, path, url = self.initialize_model_from_stub(self.source)
+            _model_args = self.initialize_model_from_stub(stub=self.source)
+            files, path, url, validation_results, compressed_size = _model_args
             if download_path is not None:
                 path = download_path  # overwrite cache path with user input
         else:
             # initializing the model from the path
             files, path, url = self.initialize_model_from_directory(self.source)
+            validation_results, compressed_size = None, None
```

Comment:
```suggestion
            validation_results, compressed_size, url = None, None, None
```

And let's remove the `url` output from `self.initialize_model_from_directory`